tests/test_workers/test_pkg_managers/test_general.py

```diff
@@ -18,38 +17,3 @@ def test_update_request_with_deps(mock_requests, sample_deps_replace):
     ]
     assert mock_requests.patch.call_count == 3
     mock_requests.patch.assert_has_calls(calls)
-
-
-@mock.patch('cachito.workers.utils.get_worker_config')
-def test_add_deps_to_bundle(mock_get_worker_config, tmpdir):
```

Comment:
Why was this test removed?

Reply:
With the new `RequestBundleDir`, it is simple enough to just call `shutil.copytree` as the last step inside `resolve_gomod`.